proto/greenfield/storage/types.proto

```diff
@@ -27,17 +30,19 @@ message BucketInfo {
   int64 create_at = 6;
   // payment_address is the address of the payment account
   string payment_address = 7 [(cosmos_proto.scalar) = "cosmos.AddressString"];
-  // primary_sp_address is the address of the primary sp. Objects belongs to this bucket will never
+  // primary_sp_id is the unique id of the primary sp. Objects belongs to this bucket will never
   // leave this SP, unless you explicitly shift them to another SP.
-  string primary_sp_address = 8 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  uint32 primary_sp_id = 8;
```

Comment:
Is it possible to put this field`primary_sp_id` in the family`VirtualGroupFamily`？